config/default.yml

```diff
@@ -1,12 +1,11 @@
 ---
-RSpec:
+RSpecRails:
   Enabled: true
-  StyleGuideBaseURL: https://rspec.rubystyle.guide
-  DocumentationBaseURL: https://docs.rubocop.org/rubocop-rspec
-  Include: &1
+  DocumentationBaseURL: https://docs.rubocop.org/rubocop-rspec_rails
+  Include:
     - "**/*_spec.rb"
     - "**/spec/**/*"
-  Language: &2
+  Language:
```

Comment:
This needs to be removed. We don’t want our users (and gem authors who make their additions) to override it for both RSpec/Language and RSpecRails/Language

Reply:
Indeed it is. I remove it!